spec/stack.js

```diff
@@ -18,19 +18,52 @@ describe('Stack', () => {
       expect(() => aStack.push('foo'))
         .to.alter(() => aStack.length(), { from: 0, to: 1 })
     })
+    it('pushes an element to the top of the stack.', () => {
+      const aStack = new Stack()
+
+      aStack.push('bar')
+      aStack.push('foo')
+
+      expect(() => aStack[0]
+        .to.equal('foo'))
+    })
+
   })
 
   context('pop()', () => {
     it('removes an element from the top of the stack.', () => {
-      const aStack = new Stack( 1 )
+      const aStack = new Stack(['foo','bar'])
       expect(() => aStack.pop())
-        .to.alter(() => aStack.length(), { from: 1, to: 0 })
+        .to.alter(() => aStack.length(), { from: 2, to: 1 })
+    })
+  })
```

Comment:
You should also test to make sure that the popped element is returned.  That is, make sure that the element on top of the stack (the element most recently pushed) is the one that is returned by `aStack.pop()`.